### ISSUE TITLE
Instead of hiding the "browse hangouts" behind the logged-in view, make it public

### ISSUE THREAD

**Issue by lpatmo** [codebuddiesdotorg/codebuddies#225]
Proposal:
- [x] 1) Add a "Browse" link to the top of the header menu.
- [x] 2) Allow visitors to click on it to see the logged-in view of the home page.
- [x] 3) If a non-logged-in visitor tries to take an action on the home page, they'll get a popup asking them to log in. 

At the moment our `templates/home/home.html` contains this logic:

```
<div id="home" class="{{#if currentUser}}logged-in{{/if}}">
  {{#unless currentUser}}
      {{> homeLoggedOut}}
  {{else}}
```

We'll have to change logic a bit so that the `home` template and the `home-logged-out` template refer to different URLs. Happy to think over/discuss implementation with another collaborator.

**Comment by lpatmo** [comment]
The advantages to making the logged-in view public, IMO:
- less mystery for new visitors who might wonder how the hangouts + the learnings section work
- more incentive to sign up to participate, maybe?

One note: I'm not sure how the "X users online learning right now" list will show anonymous users; will have to test that out.

**Comment by adachiu** [comment]
@lpatmo Instead of having the "browse" link on header menu to open up a different page, would it be simpler if we replaced the homepage's "join hangouts" screenshot with the logged-in cards view (minus the "Today I Learned" sidebar"). This way new users will get a glimpse immediately when they first arrive to the home page.

Side note: that image is also now outdated since the UI is going to be cards. We should probably replace it anyway to reflect the actual appearance of the hangout cards.

![screen shot 2016-09-28 at 1 50 58 pm](https://cloud.githubusercontent.com/assets/13413986/18931823/a2aa0126-8582-11e6-81c1-47cfb444b85b.png)

**Comment by lpatmo** [comment]
@adachiu Code-wise it might actually take a few more steps to dynamically load in the hangouts here in place of the image, but I'm not against the idea of replacing this image with loaded-in posts. :) 

My concern is whether we want new visitors to see:
- that we *do* have a learnings tab
- that there's a "What did you learn?" area underneath each hangout (this feature isn't implemented yet, but on the roadmap)
- the full history of hangouts that have happened, not just the most recent 6 (since this area is limited in dimension)

What are your thoughts?

**Comment by adachiu** [comment]
@lpatmo What if we showed a static image of the hangout cards and have a button to `View all hangouts` which brings the user to view the logged-in home? This would give a glimpse of the hangout cards on the home page. Doing this is probably easier to implement; save those steps.

In addition, below the hangout cards section, we can showcase the features `Share what you're working on` and `Today I learned` ... Example in screenshot below

![home-idea](https://cloud.githubusercontent.com/assets/13413986/18973914/ef2bac8e-8654-11e6-97f5-dce87d90dd82.jpg)
